Grants Page: add links to cards

barbaraperic opened the pull request:
## Note
- [Slack convo here ](https://filecoinfoundation.slack.com/archives/C06MTEQ3SP6/p1718038788656819)
- [Ticket](https://www.notion.so/filecoin/FF-V4-Grants-Page-Link-Grants-and-Funding-Opportunities-Cards-00b37a6254154fa98cdcbae9e6f6e85b?pvs=4)

CharlyMartin reviewed (approved):
Looking good :)

I added 2 comments to reorder/remove import statements.

Also one non-blocking question: Would it make sense to link the `Requests for Proposals` to [relevant section](https://github.com/filecoin-project/devgrants?tab=readme-ov-file#requests-for-proposals-rfps) on the `devgrants` repo?

barbaraperic commented:
> Looking good :)
> 
> I added 2 comments to reorder/remove import statements.
> 
> Also one non-blocking question: Would it make sense to link the `Requests for Proposals` to [relevant section](https://github.com/filecoin-project/devgrants?tab=readme-ov-file#requests-for-proposals-rfps) on the `devgrants` repo?

Yes, agree! fixed ✅
